### PR TITLE
Enable color in dev log

### PR DESCRIPTION
**Pull request by Velenir**
Just a small quality of life change. Turns on log coloring when project is started in development.
Great work, btw. I feel I have finally found the boilerplate to stick with.

screenshot from 2017-07-28 22 02 14

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ if (DEV) {
   const compiler = webpack([clientConfig, serverConfig])
   const clientCompiler = compiler.compilers[0]
```
Comment (faceyspacey) [inline server/index.js]: cool, u mind adding const options = { publicPath, stats: { colors: true } }

and making app.use one line again. ...trying to make the demo as easy to read as possible.

**Comment by Velenir** [comment]
Done. I think it was **prettier** that separated `app.use` into multiple lines.

File: server/index.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ if (DEV) {
...
 
-  app.use(webpackDevMiddleware(compiler, { publicPath }))
+  const options = { publicPath, stats: { colors: true } }
```
Comment (faceyspacey) [inline server/index.js]: one last thing, put the const with the other 2 conts

the idea in terms of readability is just that you see 3 app.uses in a row, and they all look the same.

putting the consts together also has the same effect.

it's small, but it's just how i do things.

**Comment by faceyspacey** [comment]
yea, it was.

File: server/index.js (hunk 2 above)
Comment (Velenir) [inline server/index.js]: Ok. Makes sense

**Comment by faceyspacey** [comment]
Thanks for this! How are things going by the way? Are you using the new webpack fork/pr? 

...im about to edit the docs and link to this:

https://www.npmjs.com/package/webpack-universal

as well as alias it in the package. to make things simpler before the medium article release.

**Comment by Velenir** [comment]
Things are great. I am really looking forward to writing more quality SSR.

I am using the fork of the original **webpack**. Works quite well but having it in npm will be even better. 
Is that what **webpack-universal** is? A fork with resolveWeak pr published?
That's great then!

**Comment by faceyspacey** [comment]
yea...feel free to use it. just open up package.json and change the special _moduleAlias key val to:
`"webpack": "webpack-universa"`

**Comment by Velenir** [comment]
Strange. It throws for me:
```
Error: Cannot find module '/home/velenir/WebstormProjects/Webpack 3/universal-demo/webpack-universal'
```
with just `"webpack": "webpack-universal"`.
I have to put `"webpack": "node_modules/webpack-universal"` for aliasing to work.
I wonder if it is a problem with **module-alias**. I thought it linked to node_modules by default.

**Comment by faceyspacey** [comment]
good find. i still haven't applied it myself.

**Pull request merged**